advanced_visualizer/requirements.txt

```diff
@@ -0,0 +1 @@
+Jinja2>=2.11.3
```

Comment:
This is not actually needed here, it can be put inside pyproject.toml file as a dependency, but it is not important now